### DISCUSSION TITLE
How to set homepage?

### DISCUSSION THREAD

**Discussion by robgrzel** [mmistakes/minimal-mistakes#2952] [Q&A] [answered]
Hi!
Could You tell me how to change homepage of my blog so it will look like : https://mmistakes.github.io/minimal-mistakes/

As of now it looks like:
https://robgrzel.github.io

**Comment by mmistakes** [comment, marked as answer]
Use [`layout: splash`](https://mmistakes.github.io/minimal-mistakes/docs/layouts/#splash-page-layout)

Here is the [source for MM splash page](https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_pages/home.md) if you need to see how it's built.

**Comment by robgrzel** [comment]
Thanks! 
Should I just copy content of home.md to index.html then change it to .md?

**Comment by mmistakes** [comment]
Yeah that should work.

**Comment by neerrrajj** [comment]
> Thanks!
> Should I just copy content of home.md to index.html then change it to .md?

Could you help me on how to do that?
